keymanapp/keyman
# chore(ios): Set transparent background on "more" icons

Fixes #6414 
This sets the background of the 3-dot "more" icon transparent.

## User Testing
Setup to match configuration in the reported issue
Device: iPhone 11 simulator
OS: iOS 15.4
Install PR build of "Keyman for iPhone and iPad"

* **TEST_NORMAL** - Test the appearance in normal mode (not dark mode)
1. Start Keyman
2. Observe the icons in the menu bar
3. Verify the 3-dot "more" icon appears and the background matches the toolbar color

* **TEST_DARK_MODE** - Test the appearance in dark mode
1. Set the device to dark mode
2. Start Keyman
3. Observe the icons in the menu bar
4. Verify the 3-dot "more" icon appears and the background matches the toolbar color
